src/layers/ZoneLayer/ZoneLayer.js

```diff
@@ -218,7 +232,17 @@ class ZoneLayer extends CasaLayer {
       this.abortController.abort();
     }
 
-    this.olLayer.getSource().clear();
+    this.featuresLayer.getSource().clear();
+    this.labelsLayer.getSource().clear();
+  }
+
+  /**
+   * Returns available layers in the layer group
+   * @private
+   * @inheritdoc
+   */
+  getLayers() {
```

Comment:
Where do you use it ?

Reply:
I kept in in the code to be able to access the layers from the layergroup from outside. I can remove it if it doesn't seem useful